src/axom/sidre/core/MFEMSidreDataCollection.hpp

```diff
@@ -602,6 +598,9 @@ class MFEMSidreDataCollection : public mfem::DataCollection
   // Used as part of Load()
   void reconstructMesh();
 
+  // Reconstructs a single field from its corresponding Sidre group
+  void reconstructField(Group* field_grp);
+
   // Reconstructs all non-mesh-related fields using the current contents
   // of the datastore, used as part of Load()
```

Comment:
Suggestion: Please document the expectation that the nodal grid function has already been reconstructed/registered